solr/core/src/java/org/apache/solr/storage/CompressingDirectory.java

```diff
@@ -167,6 +170,12 @@ public long fileLength(String name) throws IOException {
     return readLengthFromHeader(path);
   }
 
+  @Override
+  public long onDiskFileLength(String name) throws IOException {
+    Path path = directoryPath.resolve(name);
+    return FileUtils.sizeOf(path.toFile());
```

Comment:
We don't need `FileUtils` here -- prefer `Files.size(directoryPath.resolve(name))` (as is done in `FSDirectory.fileLength()`)
`